composer.json

```
@@ -14,7 +14,8 @@
     "php": "^8.3",
     "ext-gd": "*",
     "ext-dom": "*",
-    "sculpin/sculpin": "3.3.0-alpha"
+    "sculpin/sculpin": "3.3.0-alpha",
```

```suggestion
    "sculpin/sculpin": "3.3.0-alpha3",
```

We can try to use 3.3.0-alpha3, but maybe upgrade it in separate PR

---

Yeah, separate pr would be better